azkaban-common/src/main/java/azkaban/executor/ExecutableFlow.java

```diff
@@ -359,6 +372,15 @@ public void fillExecutableFromMapObject(
       this.slaOptionStr = slaBuilder.toString();
     }
 
+    if (flowObj.containsKey(VERSIONSET_JSON_PARAM) && flowObj.containsKey(VERSIONSET_MD5HEX_PARAM) && flowObj.containsKey(VERSIONSET_ID_PARAM)) {
+      final String versionSetJsonString = flowObj.getString(VERSIONSET_JSON_PARAM);
+      final String versionSetMd5Hex = flowObj.getString(VERSIONSET_MD5HEX_PARAM);
+      final int versionSetId = flowObj.getInt(VERSIONSET_ID_PARAM);
+      final VersionSet versionSet = new VersionSet(versionSetJsonString, versionSetMd5Hex,
+          versionSetId);
+      this.setVersionSet(versionSet);
```

Comment:
in our coding guidelines, I believe we don't use keyword "this" with class methods. Please check and make changes if needed.